scripts/release.py

```diff
@@ -0,0 +1,199 @@
+import json
+import os
+import re
+import subprocess
+
+from dotenv import load_dotenv
+from github import Github
+
+
+"""This release notes script is built to create a release notes draft for release candidates, patches, and minor releases.
+
+Setup:
+1. Create a `.env` file in the scripts directory.
+2. Create Github token: https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token#creating-a-personal-access-token-classic
+3. Give the Github token repo, user, audit_log, and project permissions.
+2. Add `export GH_TOKEN=<github token>` to the `.env` file.
+
+Usage:
+Required:
+    BASE - The base branch you are building your release candidate, patch, or minor release off of.
+        If this is a rc1, then just specify the branch you'll create after the release is published. e.g. BASE=1.9
+
+Optional:
```

Comment:
Could we add an optional param to generate release notes without creating a draft release? I could see it being useful instead of manually using reno. If this is out of scope, feel free to ignore.

Reply:
Sure, I'll add that.